src/reporter.js

```diff
@@ -36,25 +36,45 @@ var resultsFolder;
  * @returns {testcase record object}
  */
 
-function createTestRecord(test) {
+function loadConfiguration(options) {
+
+  console.debug('START: configuration & options:');
+
+  resultsFolder = 'results';
+  logsFolder = path.join('cypress', 'logs');
+
+  const CONFIG_FILE = path.join(os.tmpdir(), "cxr-cypress.config.json");
+  if (! fs.existsSync(CONFIG_FILE) ) {
+    console.error("PROBLEM: This reporter requires to be configured as a plugin in 'cypress.config.js'");
```

Comment:
You need to `throw`, not error-log. Otherwise there would be a crash further down.